### PR TITLE
Add additional test vectors for Prio3

### LINKED ISSUE #312: Choose repo revision for test vector reference
We removed them a couple of drafts ago because updating them was becoming a pain. (Note that machine-readable test vectors can be found in https://github.com/cfrg/draft-irtf-cfrg-vdaf/tree/main/poc/test_vec.)

### PR DESCRIPTION
**Pull request by cjpatton**
Partially addresses #312 (just need to update the hash and remove the TODO in the test vectors section).

* Exercise aggregation of multiple measurements.
* Test the range boundaries, e.g., `max_measurement` for Prio3Sum and `max_weight` for Prio3MultihotCountVec.
* Exercise more chunk lengths for variants using ParallelSum.

### CODE REVIEW

File: test_vec/vdaf/Prio3Sum_2.json [hunk 1]
```diff
@@ -0,0 +1,230 @@
+{
+    "agg_param": "",
+    "agg_result": 1521,
+    "agg_shares": [
+        "3d90b1d19acf1386",
+        "b5754e2e6430ec79"
+    ],
+    "ctx": "736f6d65206170706c69636174696f6e",
+    "max_measurement": 22,
```
Comment (divergentdave) [inline test_vec/vdaf/Prio3Sum_2.json:9]: This looks wrong, something fishy is happening.

Comment (divergentdave) [reply test_vec/vdaf/Prio3Sum_2.json:9]: Ah, `test_vec_set_type_param()` on `Sum` is just grabbing the wrong field to store in the test vector.

Comment (cjpatton) [reply test_vec/vdaf/Prio3Sum_2.json:9]: GREAT GRAND (will fix0

Comment (cjpatton) [reply test_vec/vdaf/Prio3Sum_2.json]: Done, nice catch.

**Review by divergentdave — APPROVED** [review]

**Pull request merged**